tox.ini

```diff
@@ -43,14 +44,14 @@ whitelist_externals = bash
 skip_install = True
 deps =
     flake8
-    flake8-import-order==0.9
+    flake8-import-order>=0.9
 
 [testenv:py27-lint-imports-include-list]
 commands = bash .ci/flake8_wrapper_imports.sh
 whitelist_externals = bash
 skip_install = True
 deps =
-    flake8==3.0.4
+    flake8==3.2.1
```

Comment:
I think this is causing the failure of the corresponding Travis job with lots of errors that I fixed in 17.01, revert?

Reply:
The tests won't run at all with older versions, how about we add `I201` to the exclude list for 16.10?